### ISSUE TITLE
Build failure: atlauncher

### ISSUE THREAD

**Issue by GetPsyched** [NixOS/nixpkgs#356259]
## Steps To Reproduce

1. build `atlauncher`

## Build log

<details>

<summary>Build Log</summary>

```
atlauncher> FAILURE: Build completed with 2 failures.
atlauncher> 1: Task failed with an exception.
atlauncher> -----------
atlauncher> * Where:
atlauncher> Build file '/build/source/build.gradle' line: 325
atlauncher> * What went wrong:
atlauncher> A problem occurred evaluating root project 'ATLauncher'.
atlauncher> > Could not create task ':createExe'.
atlauncher>    > Could not create task of type 'Launch4jLibraryTask'.
atlauncher>       > Could not resolve all files for configuration ':launch4jBin'.
atlauncher>          > Could not find launch4j-3.50-workdir-linux.jar (net.sf.launch4j:launch4j:3.50).
atlauncher>            Searched in the following locations:
atlauncher>                https://repo.maven.apache.org/maven2/net/sf/launch4j/launch4j/3.50/launch4j-3.50-workdir-linux.jar
atlauncher> * Try:
atlauncher> > Run with --stacktrace option to get the stack trace.
atlauncher> > Run with --info or --debug option to get more log output.
atlauncher> > Run with --scan to get full insights.
atlauncher> > Get more help at https://help.gradle.org.
atlauncher> ==============================================================================
atlauncher> 2: Task failed with an exception.
atlauncher> -----------
atlauncher> * Where:
atlauncher> Build file '/build/source/build.gradle' line: 325
atlauncher> * What went wrong:
atlauncher> A problem occurred configuring root project 'ATLauncher'.
atlauncher> > Could not create task ':createExe'.
atlauncher>    > Could not create task of type 'Launch4jLibraryTask'.
atlauncher>       > Could not resolve all files for configuration ':launch4jBin'.
atlauncher>          > Could not find launch4j-3.50-workdir-linux.jar (net.sf.launch4j:launch4j:3.50).
atlauncher>            Searched in the following locations:
atlauncher>                https://repo.maven.apache.org/maven2/net/sf/launch4j/launch4j/3.50/launch4j-3.50-workdir-linux.jar
atlauncher> * Try:
atlauncher> > Run with --stacktrace option to get the stack trace.
atlauncher> > Run with --info or --debug option to get more log output.
atlauncher> > Run with --scan to get full insights.
atlauncher> > Get more help at https://help.gradle.org.
atlauncher> ==============================================================================
atlauncher> Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.
atlauncher> You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.
atlauncher> For more on this, please refer to https://docs.gradle.org/8.10.2/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
atlauncher> BUILD FAILED in 2m 31s
```

</details>

## Additional context

TL;DR: Launch4j attempts to run the `createExe` task, which is only relevant to Windows. This _should_ be skipped given it's explicitly marked to be skipped, but that does not work. I've asked the upstream maintainer regarding this and he's mentioned that they'll eventually move out of Launch4j (but this remains a nix derivation failure and not an upstream issue).

## Metadata

- system: `"x86_64-linux"`
- host os: `Linux 6.11.7, NixOS, 24.11 (Vicuna), 24.11.20241109.76612b1`
- multi-user?: `yes`
- sandbox: `yes`
- version: `nix-env (Nix) 2.24.10`
- nixpkgs: `/nix/store/0sxsfzswhjck7f4sbsvznz285s4i7y64-source`

## Notify maintainers

`@me` :)

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by GetPsyched** [comment]
Why was this closed?

**Comment by GetPsyched** [comment]
This was fixed a while back, and more recently, launch4j was removed entirely
